test/fixtures/escape-strings/style.scss

```diff
@@ -0,0 +1,26 @@
+@import 'variables.json';
+
+/// Map deep get
+/// @author Hugo Giraudel
+/// @access public
+/// @param {Map} $map - Map
+/// @param {Arglist} $keys - Key chain
+/// @return {*} - Desired value
+@function map-deep-get($map, $keys...) {
```

Comment:
I like the real-world test, but all of this SASS logic kind of obscures what's actually being tested. I think we should be able to simplify this a great deal to really get at what it is that we want to test. Review #5 for some simpler examples, like @diddledan's [original example](https://github.com/Updater/node-sass-json-importer/pull/5#issuecomment-149422978) or @safareli's [list of edge cases](https://github.com/Updater/node-sass-json-importer/issues/7#issuecomment-154046606).